File: generators/ratings.py
#INSERT INTO ratings (user_id, title_id, does_recommand) VALUES (1, 1, true); 
import random

f = open("ratingsList.txt", "a")
for i in range(0,100,1):
    random.seed()
    nbrReview = random.randrange(8) + 3
    for j in range(0,nbrReview,1):
        reviewedFilm = random.randrange(5850)
        f.write("INSERT INTO ratings (user_id, title_id, does_recommend) VALUES ({}, {}, {});\n".format(i, reviewedFilm, random.random() > 0.5))
f.close()

File: Streamify/Models/Ratings.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Streamify;


[Table("ratings")]
public class Ratings {
    [Key]
    [Column("user_id")]
    public Int UserId { get; set; }

    [Key]
    [Column("title_id")]
    public Int TitleId { get; set; }

    [Column("does_recommend")]
    public Boolean DoesRecommend { get; set; }
}